Created events

emico7 opened the pull request:
My current test coverage is barely above 80%. I will add tests for the api controller in the next checkpoint. @wonderwoman13

emico7 committed:
Created events

brittanyjspringer commented:
@emico7 can you please remove the empty asset files and add your log folder to your .gitignore? Thank you!

emico7 committed:
Deleted unused files in app/assets

emico7 committed:
Added log dir to .gitignore

emico7 commented:
@wonderwoman13 Just wanted to double check -- for adding the log folder to .gitignore, I added a line `/log/` in .gitignore file. Was that all I needed to do? Also, I deleted all the empty files in the asset. When did all those files get created?

brittanyjspringer commented:
Yes, it was! Merging now 👍
